modules/badge/css/package.json

```
@@ -0,0 +1,33 @@
+
+{
+  "name": "@workday/canvas-kit-css-badge",
+  "version": "0.0.0",
+  "description": "provides a summary indicator with dynamic values",
+  "author": "Workday, Inc. (https://www.workday.com)",
+  "license": "Apache-2.0",
+  "files": [
+    "dist",
+    "lib",
+    "index.scss"
+  ],
+  "style": "dist/canvas-kit-css-badge.min.css",
+  "main": "dist/canvas-kit-css-badge.min.css",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Workday/canvas-kit/tree/master/modules/badge/css"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "node ../../../../utils/css-build.js index.scss"
```

```suggestion
    "build": "node ../../../utils/css-build.js index.scss"
```
It goes one level too far back.